pkg/engine2/operational_eval/vertex_property_test.go

```diff
@@ -113,3 +113,103 @@ func Test_propertyVertex_evaluateEdgeOperational(t *testing.T) {
 		})
 	}
 }
+
+func Test_propertyVertex_Dependencies(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	dcap := NewMockdependencyCapturer(ctrl)
```

Comment:
Should probably not share mocks across tests otherwise they could have implicit dependencies (eg, case 2 only works when run serially after case 1 runs its `mocks()`). IMO, change the test to `mocks func(*MockdependencyCapturer)` and create the mock per-test in the `Run`.

Reply:
ctrl.Finish() is supposed to rest mocks, but i can move this